templates/local.conf.append

```diff
@@ -1,71 +0,0 @@
-
```

Comment:
It's not clear to me from the commit text why any of the files in `templates` should be removed.